ocsp_asn1crypto.py

```diff
@@ -28,6 +29,12 @@
 from snowflake.connector.ssd_internal_keys import ret_wildcard_hkey
 from os import getenv
 
+# force versioned dylibs onto oscrypto ssl on catalina
+if sys.platform == 'darwin' and platform.mac_ver()[0].startswith('10.15'):
+    from oscrypto import use_openssl
+    use_openssl(libcrypto_path='/usr/lib/libcrypto.35.dylib', libssl_path='/usr/lib/libssl.35.dylib')
```

Comment:
ok, so Mojave has this:
```
> ls -l /usr/lib/libssl*
-rwxr-xr-x  1 root  wheel  392912 Jul 29 13:59 /usr/lib/libssl.0.9.7.dylib
-rwxr-xr-x  1 root  wheel  642464 Jul 29 13:59 /usr/lib/libssl.0.9.8.dylib
-rwxr-xr-x  1 root  wheel  624624 Jul 29 13:59 /usr/lib/libssl.35.dylib
-rwxr-xr-x  1 root  wheel  599888 Jul 29 13:59 /usr/lib/libssl.43.dylib
-rwxr-xr-x  1 root  wheel  569984 Jul 29 13:59 /usr/lib/libssl.44.dylib
lrwxr-xr-x  1 root  wheel      15 Aug  6 14:42 /usr/lib/libssl.dylib -> libssl.35.dylib
```

but Catalina has this:
```% ls -l /usr/lib/libssl*
-rwxr-xr-x  1 root  wheel  212288 Oct 23 17:40 /usr/lib/libssl.0.9.7.dylib
-rwxr-xr-x  1 root  wheel  335888 Oct 23 17:40 /usr/lib/libssl.0.9.8.dylib
-rwxr-xr-x  1 root  wheel  330592 Oct 23 17:40 /usr/lib/libssl.35.dylib
-rwxr-xr-x  1 root  wheel  314032 Oct 23 17:40 /usr/lib/libssl.43.dylib
-rwxr-xr-x  1 root  wheel  300512 Oct 23 17:40 /usr/lib/libssl.44.dylib
-rwxr-xr-x  1 root  wheel  294048 Oct 23 17:40 /usr/lib/libssl.46.dylib
-rwxr-xr-x  1 root  wheel   32976 Oct 23 17:40 /usr/lib/libssl.dylib
```

and Sierra:
```
>ls -l /usr/lib/libssl.*
-rwxr-xr-x  1 root  wheel  396912 Jul 14  2017 /usr/lib/libssl.0.9.7.dylib*
-rwxr-xr-x  1 root  wheel  646416 Sep 21 13:41 /usr/lib/libssl.0.9.8.dylib*
-rw-r--r--  1 root  wheel  942880 Sep 21 13:41 /usr/lib/libssl.35.dylib
-rw-r--r--  1 root  wheel  882656 Sep 21 13:41 /usr/lib/libssl.39.dylib
lrwxr-xr-x  1 root  wheel      18 Dec  3  2018 /usr/lib/libssl.dylib@ -> libssl.0.9.8.dylib
```

I'm not sure what the number is, though, libssl35 looks most common ones. (different sizes)